test/Common.h

```diff
@@ -24,18 +24,28 @@
 #include <boost/noncopyable.hpp>
 #include <boost/program_options.hpp>
 
+#include <vector>
+
 namespace solidity::test
 {
 
+class EVMHost;
+
 #ifdef _WIN32
 static constexpr auto evmoneFilename = "evmone.dll";
 static constexpr auto evmoneDownloadLink = "https://github.com/ethereum/evmone/releases/download/v0.4.1/evmone-0.4.1-windows-amd64.zip";
+static constexpr auto heraFilename = "hera.dll";
```

Comment:
I would suggest this refactoring PR only makes support for multiple EVMC VMs, but only keeps evmone. Then separately we can introduce hera and running ewasm tests.

Reply:
Also hera 0.2.5 is not evmc7 compatible, 0.2.6 will be :)

Reply:
ah yes, I know.. I just wanted to have the hera things similar to the evmone things.. but I where also thinking to dont introduce any specific hera related code.. right now additional evmc vms can be defined by setting `--evmc <path-to-a-evmc-library>`  multiple times